src/platform/baytrail/platform.c

```diff
@@ -58,6 +59,12 @@ static const struct sof_ipc_fw_ready ready
 #endif
 		.tag = SOF_TAG,
 		.abi_version = SOF_ABI_VERSION,
+		.compiler = {
+			.micro = CC_MICRO,
+			.minor = CC_MINOR,
+			.major = CC_MAJOR,
+			.name = CC_NAME,
+		}
 	},
```

Comment:
This looks like ABI change, isn't it? And it requires some changes on kernel side too.